oval_graph/arf_to_json.py

```diff
@@ -4,14 +4,26 @@
 import shutil
 import pprint
 from datetime import datetime
-
+import sys
 
 from .converter import Converter
 from .client import Client
 
 
 class ArfToJson(Client):
 
+    def run_gui_and_return_answers(self):
```

Comment:
It would be good to have a possibility to disable the `inquirer` even if it is installed. Utilities s.a. `ls` or `grep` also allow to switch off color output, although they can detect whether it is a good idea to show colors or not.

Reply:
I plan to add an option to deactivate the selection that will process all the matching rules.